app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h2><%= @recipe.name =%></h2>
+<p>Complexity: <%= @recipe.complexity %></p>
+<p>Genre: <%= @recipe.genre %></p>
+<p>Ingredients: <% @recipe.ingredients.each do |ingredient| %>
+    <li><%= ingredient.name %></li>
+    <% end %>
+<p>Total Cost: <%= @recipe.total_cost %></p>
+
+<h1>Add Ingredient:</h1>
+<%= form_with url: "/recipes/#{@recipe.id}", method: :patch, data: {turbo: false} do |f| %>
```

Comment:
This is where you could consider the path to be `/recipes/:recipe_id/ingredients` and be a POST instead of patch. That's a slightly more RESTful option